experimental/traffic-portal/src/app/core/cdns/cdn-table/cdn-table.component.html

```diff
@@ -0,0 +1,28 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<mat-card class="table-page-content">
+	<div class="search-container">
+		<input type="search" name="fuzzControl" aria-label="Fuzzy Search CDNs" rowSelection="single" autofocus inputmode="search" role="search" accesskey="/" placeholder="Fuzzy Search" [formControl]="fuzzControl" (input)="updateURL()"/>
+	</div>
+	<tp-generic-table
+		[data]="cdns | async"
+		[cols]="columnDefs"
+		[fuzzySearch]="fuzzySubject"
+		context="cdns"
+		[contextMenuItems]="contextMenuItems"
+		(contextMenuAction)="handleContextMenu($event)">
+	</tp-generic-table>
+</mat-card>
+
+<button class="page-fab" mat-fab title="Create a new CDN" *ngIf="auth.hasPermission('CDN:CREATE')" routerLink="new"><mat-icon>add</mat-icon></button>
```

Comment:
navigation should be handled by Anchor tags (`<a>`), not buttons

Reply:
Using an anchor in 16ab99ecba